packages/grid/src/vaadin-grid.js

```diff
@@ -268,11 +269,19 @@ class Grid extends GridMixin(ElementMixin(ThemableMixin(ControllerMixin(PolymerE
         ios$="[[_ios]]"
         loading$="[[loading]]"
         column-reordering-allowed$="[[columnReorderingAllowed]]"
+        show-empty-state$="[[__showEmptyState]]"
```

Comment:
Is there a reason for using `show-` in the attribute name? Could it be just `empty-state`?

Reply:
Renamed the attribute and the property